### PR TITLE
Link Advanced Keda Example

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- link keda example in keda-manager repo
- disable istio side car injection in keda workloads as sidecar injection of Istio prevents the api service of KEDA to work properly (prometheus scaler didn't work) [troubleshooting](https://keda.sh/docs/2.10/troubleshooting/)

**Related issue(s)**
#80

### CODE REVIEW

**Comment by kwiatekus** [comment]
waits for https://github.com/kyma-project/keda-manager/pull/156

**Review by pPrecel — APPROVED** [review]

File: docs/user/04-20-demo-applications.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# KEDA demo application
+## Keda CPU Scaler Example
```
Comment (grego952) [inline docs/user/04-20-demo-applications.md:1]:
```suggestion
## Keda CPU scaler example
```

File: docs/user/04-20-demo-applications.md [hunk 2]
```diff
@@ -37,3 +37,17 @@ kubectl get hpa -n keda-demo
 NAMESPACE   NAME                                 REFERENCE                   TARGETS   MINPODS   MAXPODS   REPLICAS   AGE
 keda-demo   keda-hpa-orders-service-cpu-scaler   Deployment/orders-service   80%/30%   1         10        4          31s
 ```
+
+## Keda Prometheus Scaler Example
```
Comment (grego952) [inline docs/user/04-20-demo-applications.md:41]:
```suggestion
## Keda Prometheus scaler example
```

File: docs/user/04-20-demo-applications.md [hunk 3]
```diff
@@ -37,3 +37,17 @@ kubectl get hpa -n keda-demo
...
+
+Please follow [the linked example](https://github.com/kyma-project/examples/tree/main/scale-to-zero-with-keda) to experience how Kyma Keda module can complement other Kyma components.
```
Comment (grego952) [inline docs/user/04-20-demo-applications.md:43]:
```suggestion
Follow [this example](https://github.com/kyma-project/examples/tree/main/scale-to-zero-with-keda) to experience how Kyma's Keda module can complement other Kyma components.
```

File: docs/user/04-20-demo-applications.md [hunk 4]
```diff
@@ -37,3 +37,17 @@ kubectl get hpa -n keda-demo
...
+
+It demonstrates an event-driven approach that allows you to decouple functional parts of an application and apply consumption-based scaling.
+
+It uses: 
+ - Functions to deploy workloads directly from a Git repository (kyma serverless),
```
Comment (grego952) [inline docs/user/04-20-demo-applications.md:48]:
```suggestion
 - Functions to deploy workloads directly from a Git repository ([Kyma Serverless](https://kyma-project.io/docs/kyma/latest/01-overview/main-areas/serverless/)),
```

File: docs/user/04-20-demo-applications.md [hunk 5]
```diff
@@ -37,3 +37,17 @@ kubectl get hpa -n keda-demo
...
+ - In-cluster Eventing to enable event-driven communication (kyma eventing),
```
Comment (grego952) [inline docs/user/04-20-demo-applications.md:49]:
```suggestion
 - In-cluster Eventing to enable event-driven communication ([Kyma Eventing](https://kyma-project.io/docs/kyma/latest/01-overview/main-areas/eventing/)), 
```

**Review by grego952 — APPROVED** [review]

**Review by Cortey — APPROVED** [review]

**Pull request merged**